### ISSUE TITLE
Testing for blue dotted line bouncing

### ISSUE THREAD

**Issue by jjlromero** [jjlromero/r-test-repo-1#2]
Ticket 2084898  says that when reaching the last line of making a comment on an issue, the blue dotted lines will start to go crazy and start to bounce. I just hope I can type enough to see if that happens. I don't thing it will happen - I think it was just something that was wonky with the system that day.

**Comment by john89521** [comment]
Going to try to replicate this problem but I am not sure I will be able to since it seems like it was just a fluke to happen on that day and on that user's computer. 
The steps say you have to type on every line of the editor. 
The last line is supposed to be the wonky line - the user says that there will appear blue dotted lines that start to bounce and go crazy. However, my experience is that it just moved to the last line and I believe that is what it is supposed to do. So far this is proving to be okay. 
Going to type a few more.

**Comment by john89521** [comment]
I decided to try one more time to get that blue dotted line to appear. 
I don't think it is going to appear.
I think it was just a wonky item that appeared that day. 
I think it was just a wonky item that appeared that day. I actually like it when I can reproduce an error. 
There is a strange satisfaction when the user writes in and then I see the same thing - as opposed to this one where the error is not happening.

**Comment by john89521** [comment]
I decided to try one more time to get that blue dotted line to appear.
I don't think it is going to appear.
I think it was just a wonky item that appeared that day.
I think it was just a wonky item that appeared that day. I actually like it when I can reproduce an error.
There is a strange satisfaction when the user writes in and then I see the same thing - as opposed to this one where the error is not happening.
<img width="1164" alt="Screenshot 2023-04-05 at 8 28 07 AM" src="https://user-images.githubusercontent.com/13544800/230625895-38c31086-5f9a-4e24-8c08-d000af1da0e9.png">
I just attached and image to see if this think will work. I hope I can delete these comments after I test this entire process because it is getting long. Well, actually, after doing that, the guy has some points. The line is bouncing up and down and it is sort of weird. I will now post the comment to see if the image comes into the window.
